frontend/src/store/modules/project.ts

```diff
@@ -57,6 +57,11 @@ function convert(
     }
   }
 
+  // sort the member list
+  memberList.sort((a, b) => {
```

Comment:
Do we still need to do this since we have already done this in the modules/

Reply:
It indeed is. 
These two lists are different from each other. 
project.ts deal with ProjectMember, in the meanwhile member.ts deal with Member